### PR TITLE
feat: add final message

### PR DESCRIPTION
**Pull request by j-zimnowoda**
https://app.zenhub.com/workspaces/dev-5e7e84dddc966f05a627a19b/issues/redkubes/unassigned-issues/346

Providing command for accessing password is still problematic - I am still working on it

### CODE REVIEW

File: src/cmd/commit.ts [hunk 1]
```diff
@@ -154,7 +154,20 @@ export const commit = async (): Promise<void> => {
   if (values?.charts?.gitea?.enabled) await commitAndPush()
   else d.log('The files have been prepared, but you have to commit and push to the remote yourself.')
 
-  if (isChart) await setDeploymentStatus()
+  if (isChart) {
+    await setDeploymentStatus()
+    const credentails = values!.charts.keycloak
```
Comment (Morriz) [inline src/cmd/commit.ts:159]: if the word "credentails" is not caught with spellcheck, then is that not running in your environment?

File: src/cmd/commit.ts [hunk 2]
```diff
@@ -154,7 +154,20 @@ export const commit = async (): Promise<void> => {
...
+    const message = `
+    ########################################################################################################################################
+    #
+    #  To start using Otomi, first follow the post installation steps: https://otomi.io/docs/installation/post-install/ 
+    #  The URL to access Otomi Console is: https://otomi.${values!.cluster.domainSuffix}
+    #  When Keycloak is used as IDP, the URL to access Keycloak is: https://keycloak.${values!.cluster.domainSuffix}
```
Comment (Morriz) [inline src/cmd/commit.ts:165]:
```suggestion
    #  When no external IDP was configured, please log into Keycloak first to create one or more users and add them either to the `team-admin` or `admin` group.
    # The URL to access Keycloak is: https://keycloak.${values!.cluster.domainSuffix}
```

**Review by Morriz — CHANGES_REQUESTED** [review]
I added some cosmetic changes

File: src/cmd/commit.ts (hunk 2 above)
Comment (j-zimnowoda) [reply src/cmd/commit.ts:165]: done

File: src/cmd/commit.ts (hunk 1 above)
Comment (j-zimnowoda) [reply src/cmd/commit.ts:159]: done

**Review by Morriz — APPROVED** [review]

**Pull request merged**